reactive/kubernetes_master.py

```diff
@@ -1659,12 +1659,13 @@ def configure_apiserver():
     api_opts['etcd-servers'] = etcd_connection_string
 
     # Many admission plugins are enabled by default as of 1.10.
```

Comment:
Sorry, I don't follow.

Reply:
Something like: "Admission plugins were introduced in 1.10. Here is the current list of default admission plugins as of 1.16."

The way it reads now implies to me that this list has been around since 1.10.